### ISSUE TITLE
Login page has 'Show password'

### ISSUE THREAD

**Issue by glen** [snehaq/MailSenderApplication#14]
On Login page there never is 'Show password' in a web application.
Is there any justification for having a 'Show password' check box?

**Comment by hussain33** [comment]
Yes.For user friendly nature.If the user has a long password and thinks he has made a mistake and he can use show password functionality to see his typed characters.

**Comment by hussain33** [comment]
If you feel the need then will exclude it from the login page.Waiting for your response.

**Comment by glen** [comment]
Give me examples of renowned websites where this feature is available.

**Comment by hussain33** [comment]
none.

**Comment by hussain33** [comment]
Removed the show password section from jsp and javascrpt file.
